crates/reqwest/src/lib.rs

```diff
@@ -38,7 +38,6 @@ use bytes::Bytes;
 use futures::FutureExt;
 use futures::future::BoxFuture;
 pub use http_cache_stream::X_CACHE;
-pub use http_cache_stream::X_CACHE_DIGEST;
```

Comment:
Note: it is no longer possible to add this header to the response as the digest is no longer calculated prior to returning from the middleware. The response from the middleware now has a body which transparently writes to the cache and is calculating its digest as the body is streamed.